presto-main/src/main/java/com/facebook/presto/resourcemanager/ResourceManagerResourceGroupService.java

```diff
@@ -41,6 +41,7 @@
     private final InternalNodeManager internalNodeManager;
     private final Cache<InternalNode, List<ResourceGroupRuntimeInfo>> cache;
     private final Executor executor = Executors.newCachedThreadPool();
+    private final Boolean resourceGroupServiceCacheEnable;
```

Comment:
`LoadingCache` also implements `Function`.  When the cache is disabled, should we just have the function be a method reference to `this::getResourceGroupInfos`, and when the cache is enabled, its implementation is a `LoadingCache`?

Reply:
Done